analysers/analyser_osmosis_relation_public_transport.py

```diff
@@ -406,7 +562,13 @@ def __init__(self, config, logger = None):
         self.callback80 = lambda res: {"class":8, "data":[self.relation_full, self.any_full, self.positionAsText]}
         self.callback90 = lambda res: {"class":9, "data":[self.relation_full, self.any_full, self.positionAsText]}
         self.callbackA0 = lambda res: {"class":10, "data":[self.node_full, self.positionAsText]}
+        self.callback100 = lambda res: {"class":11, "data":[self.relation_full, self.positionAsText]}
+        self.callback110 = lambda res: {"class":12, "data":[self.relation_full, self.any_full, self.positionAsText]}
 
+        if self.config.options.get("driving_side") == "left":
+            self.buffer_driving_side = "left"
+        else:
+            self.buffer_driving_side = "right"
```

Comment:
Unused variable `buffer_driving_side`.

Reply:
pff, I really need vacations ... Should be ok now